charts/asserts/templates/server/configmap.yaml

```diff
@@ -340,6 +340,11 @@ data:
       stats:
         initialDelay: 8640000000
         fixedDelay: 8640000000
+    graph:
+      trimming:
+        enable: true
+        retentionDays:
+          default: 30        
```

Comment:
30 days retention for TSDB too?

Reply:
yes, that's the default shown here: https://github.com/asserts/helm-charts/blob/master/charts/asserts/values.yaml#L648